src/test/java/org/folio/test/support/ApiTests.java

```diff
@@ -155,13 +156,25 @@ protected OkapiDeployment getOkapi() {
 
   protected void removeAllFromTable(String tableName) {
     final CompletableFuture<Void> future = new CompletableFuture<>();
-
+    Criterion criterion = new Criterion();
+    if ("feefines".equals(tableName)) {
+      for (AutomaticFeeFineType type : AutomaticFeeFineType.values()) {
+        criterion.addCriterion(createCriteria(type), "AND");
+      }
+    }
     PostgresClient.getInstance(vertx, TENANT_NAME)
-      .delete(tableName, new Criterion(), result -> future.complete(null));
+      .delete(tableName, criterion, result -> future.complete(null));
 
     get(future);
   }
 
+  private Criteria createCriteria(AutomaticFeeFineType automaticFeeFineType){
```

Comment:
Missing space. Also, let's give it a more specific name. Like "createAutomaticFeeFineExclusionCriteria"